source/api/events/command-end-event.md

```diff
@@ -0,0 +1,59 @@
+---
+title: command:end Event
+---
+
+The `command:end` event fires when cy finishes running and executing your command. Useful for debugging and understanding how commands are handled.
+
+# Environment
+
+Occurs in the **browser** and in the {% url "background process" background-process %}.
+
+# Arguments
+
+* command **(Object)**
+
+In the **browser**, the object is the actual command instance. In the **background process**, it's a serialized version of the command object.
```

Comment:
It's tricky because I don't think we really want to document the `Command` class. The whole `.get()` aspect of it is a vestige of it being a Backbone model, which I believe we're trying to move away from, but would take a lot of work and refactoring. For the most part, a command is an internal part of Cypress, but it does get exposed in some of these events. Any thoughts, @brian-mann?